server/database/index.js

```diff
@@ -1,21 +1,20 @@
 const mongoose = require('mongoose');
-var seedData = require('./seedData.json');
 
 mongoose.connect('mongodb://localhost/BTetsy', { useNewUrlParser: true, useUnifiedTopology: true })
-  .catch(error => console.log(error));
+  .catch((error) => error);
```

Comment:
Why remove the console.log? Do have another mechanism for easily telling if the server was unable to connect to the database properly?